### PR TITLE
[Merged by Bors] - feat: add Algebra.discr_localizationLocalization 

### PR DESCRIPTION
**Pull request by xroblot**
We prove
```lean
theorem Algebra.discr_localizationLocalization (b : Basis ι R S) :
  Algebra.discr Rₘ (b.localizationLocalization Rₘ M Aₘ) = algebraMap R Rₘ (Algebra.discr R b) 
```
We need for that to prove the analogue for the trace of `Algebra.norm_localization` (that was the only result in `RingTheory.Localization.Norm`). Since the results are added to this file, it is renamed to `RingTheory.Localization.NormTrace` (not sure about the name though). 

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by ericrbg** [comment]
can you make the moving of the file be one commit? maybe with `git mv`? because at the moment it's really hard to see the diff

**Comment by xroblot** [comment]
> can you make the moving of the file be one commit? maybe with `git mv`? because at the moment it's really hard to see the diff

I used your suggestion and used `git mv`. Hopefully, it will make it easier to review this PR. Thanks!

**Comment by ericrbg** [comment]
lgtm!

maintainer merge

the diff is best seen here: [`6fd5d64` (#6422)](https://github.com/leanprover-community/mathlib4/pull/6422/commits/6fd5d6409d778f9e860966b1d5336fe7776e7ca6)#

File: Mathlib/RingTheory/Localization/NormTrace.lean [hunk 1]
```diff
@@ -5,21 +5,27 @@ Authors: Anne Baanen
...
 
 #align_import ring_theory.localization.norm from "leanprover-community/mathlib"@"2e59a6de168f95d16b16d217b808a36290398c0a"
 
 /-!
 
-# Field/algebra norm and localization
+# Field/algebra norm / trace and localization
 
-This file contains results on the combination of `Algebra.norm` and `IsLocalization`.
+This file contains results on the combination of `Algebra.norm`, `Algebra.trace` and
```
Comment (riccardobrasca) [inline Mathlib/RingTheory/Localization/NormTrace.lean:16]: Can you mention here also the discriminant? Thanks!

bors d+

**Pull request closed without merging**